packages/datetime/src/dateRangeInput.tsx

```diff
@@ -512,6 +753,10 @@ export class DateRangeInput extends AbstractComponent<IDateRangeInputProps, IDat
         return (boundary === DateRangeBoundary.START) ? DateRangeBoundary.END : DateRangeBoundary.START;
     }
 
+    private areSameDay = (a: moment.Moment, b: moment.Moment) => {
```

Comment:
Worth splitting up `DateUtils` into `DateUtils` and `MomentUtils`? There have been many times I've had to check whether I'm looking at dates or moments in that file and try and work my way around name clashes.